File: java64/nivel3/exer25.java
import javax.swing.JOptionPane;

public class exer25 {
    public static void main(String[] args) {
        double altura;
        String sexo;

        altura = Double.parseDouble(JOptionPane.showInputDialog("Informe sua altura: "));
        sexo = JOptionPane.showInputDialog("Informe seu sexo: \nMasculino \nFeminino");

        switch (sexo) {
            case "Masculino":
                JOptionPane.showMessageDialog(null, "Peso ideal: " + ((72.7 * altura) - 58));
                break;
            case "Feminino":
                JOptionPane.showMessageDialog(null, "Peso ideal: " + ((62.1 * altura) - 44.7));
                break;
            default:
                break;
        }
    }
}

File: pyhton64/nivel3/exer25.py
h = float(input('Informe sua altura: '))
sexo = str(input('Sexo:\nM - masculimo\nF - feminino\nEscolha: '))

if sexo == 'M' or sexo == 'm':
    peso = (72.7 * h) - 58
    print(f'Seu peso é: {round(peso)}')

elif sexo == 'F' or sexo == 'f':
    peso = (62.1 * h) - 44.7
    print(f'Seu peso é: {round(peso)}')
else:
    print('Informe M ou F')
